roles/confluent.schema_registry/templates/schema-registry_ssl_customcerts.properties.j2

```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-kafkastore.bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}SSL://{{ host }}:{{broker.config.port}}{% endfor %}
+kafkastore.bootstrap.servers={% for host in groups['kafka'] %}{% if loop.index > 1%},{% endif %}SSL://{{ host }}:{{kafka_port}}{% endfor %}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.